src/app/models/matchups.model.ts

```diff
@@ -0,0 +1,62 @@
+export interface WeeklyMatchupInfo {
+  week: number,
+      squadMatchups: Array<{weeklyMatchup: Array<Matchups>}>,
+      active: boolean,
+      past: boolean,
+}
+
+export interface Matchups {
+  squadUID: string,
+  selectedSquads: Array<
+  {
+    category: string,
+    team: string,
+    points: number
+  }
+  >
+  squadName?: string;
+  matchupTotal: number
```

Comment:
What are we totaling here?

Reply:
@joshuaPSmith that is what I am going to use to get the total number of points from each category.